package.json

```diff
@@ -14,6 +14,7 @@
     "coverage": "vitest run --coverage"
   },
   "dependencies": {
+    "@fontsource/roboto": "^5.0.8",
     "@fontsource-variable/roboto-mono": "^5.0.14",
```

Comment:
should we remove this one?

Reply:
These two are different fonts, the latter is used for the cards in the navbar, I think we can remove them

Reply:
Yeah they provide cleaner numbers :)
`@fontsource/roboto` will probably still work